js/packages/teams-ai/src/ConversationHistory.spec.ts

```diff
@@ -58,9 +58,9 @@ describe('ConversationHistory', () => {
             ConversationHistory.clear(state);
         });
 
-        // TODO: Should it actually throw an error if there is no "conversation" member?
+        // TODO: (Discussion) Should it actually throw an error if there is no "conversation" member?
```

Comment:
@Stevenic, @aacebo, @singhk97, could you weigh in on this opinion?

Reply:
So if there's no conversation member then there's no place to store the conversation state which would be a problem as it can't manage conversation history. And exception seems appropriate